package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Base ESLint styles by Underdog.io",
   "main": "index.js",
   "scripts": {
-    "lint": "eslint -c ./index.js ./**/*.js",
+    "lint": "npm run lint-es5 && npm run lint-es6",
+    "lint-es5": "eslint -c ./index.js ./index.js ./test/index.js",
+    "lint-es6": "eslint -c ./es6.js ./**/*.js",
```

Comment:
Do we really want `./**/*.js`? That would mean we want to make sure all files in this repo conform to the `es6` rules right?

Seems like it is ok, since these are just configs and just object definitions, but would it ever cause an issue?